client/methods.tests.js

```diff
@@ -1,136 +1,66 @@
-        assert.isDefined(Package['meteor']['Meteor'].exportCoverage)
-        assert.isDefined(Package['meteor']['Meteor'].importCoverage)
-    });
 
-    it('send client coverage', function (done) {
-        try {
-            Meteor.sendCoverage(
-                function(stats,err) {
-                    assert.isTrue(stats.TOTAL > 0, "no client coverage");
-                    assert.isTrue(stats.SUCCESS > 0, "none of the client coverage have been saved");
-                    assert.isTrue(stats.FAILED === 0, "an export failed");
-                    done();
-                }
-            );
-        } catch (e) {
-            console.error (e, e.stack)
-            done(e);
-        }
+    let sandbox;
+    beforeEach(function() {
+      sandbox = sinon.sandbox.create();
+      if(global['__coverage__']) {
+        delete global['__coverage__'];
+      }
     });
 
-    it('export coverage', function (done) {
-        this.timeout(10000);
-        try {
-            Meteor.exportCoverage(
-                'coverage',
-                function(err) {
-                    assert.isUndefined(err);
-                    done();
-                }
-            );
-        } catch (e) {
-            console.error (e, e.stack)
-            done(e);
-        }
+    afterEach(function(){
+      sandbox.restore();
     });
 
-    it('import coverage', function (done) {
-        this.timeout(10000);
-        try {
-            Meteor.importCoverage(
-                function(err) {
-                    assert.isUndefined(err);
-                    done();
-                }
-            );
-        } catch (e) {
-            console.error (e, e.stack)
-            done(e);
-        }
+    it('should be defined', function () {
+        assert.isDefined(Meteor.sendCoverage);
+        assert.isDefined(Meteor.exportCoverage);
+        assert.isDefined(Meteor.importCoverage);
     });
 
-    it('export lcovonly', function (done) {
-        this.timeout(10000);
-        try {
-            Meteor.exportCoverage(
-                'lcovonly',
-                function(err) {
-                    assert.isUndefined(err);
-                    done();
-                }
-            );
-        } catch (e) {
-            console.error (e, e.stack)
-            done(e);
-        }
-    });
+    it('calls the callback function if there are no coverage', function () {
+        sandbox.stub($, 'ajax');
+        const callback = sandbox.spy();
+        Meteor.sendCoverage(callback);
 
-    it('export json', function (done) {
-        this.timeout(10000);
-        try {
-            Meteor.exportCoverage(
-                'json',
-                function(err) {
-                    assert.isUndefined(err);
-                    done();
-                }
-            );
-        } catch (e) {
-            console.error (e, e.stack)
-            done(e);
-        }
+        expect(callback).to.have.been.deep.calledWith({SUCCESS: 0, FAILED: 0, TOTAL: 0});
```

Comment:
No, this is testing for an empty `global['__coverage__']`(https://github.com/thiagodelgado111/meteor-coverage/blob/0bc69a18547d19660a7e7bdf281df4f7c2d0e7bd/client/methods.js#L24)